index.md

```diff
@@ -8,3 +8,4 @@ layout: default
 **[Hitesh Malhotra](https://github.com/CodeWithHitesh)** | Keen Learner. 
 **[Utkarsh Kansal](https://github.com/utkarshkansal17)** | Data Scientist AI Machine Learning.
 **[Firstname Lastname](https://github.com/first-last)** | Description description description.
+**[Puranjay Singh](https://github.com/purjaysin)**|Towards the top!
```

Comment:
Can you first pull the changes and than add your name, there are 13 lines in the file. If I merge your PR 2 peoples get removed from the file. just pull the changes, so you have updated files and add your line.